client/src/controllers/MessagesController.ts

```diff
@@ -0,0 +1,46 @@
+import { Controller } from '@hotwired/stimulus';
+
+export class MessagesController extends Controller<HTMLFormElement> {
+  static classes = ['newMessage', 'show'];
+  static targets = ['messages', 'templates'];
+
+  declare readonly hasMessagesTarget: boolean;
+  declare readonly hasNewMessageClass: boolean;
+  declare readonly hasShowClass: boolean;
+  declare readonly hasTemplatesTarget: boolean;
+  declare readonly messagesTarget: HTMLElement;
+  declare readonly newMessageClass: string;
+  declare readonly showClass: string;
+  declare readonly templatesTarget: HTMLElement;
+
+  add(event) {
+    const status = event.detail.status;
+    const statusText = event.detail.text;
+    // console.log("connected add.",status,statusText);
+    // console.log(this.templatesTarget);
+    if (this.hasNewMessageClass) {
+      this.element.classList.add(this.newMessageClass);
+    }
+    if (this.hasTemplatesTarget) {
+      const template = this.templatesTarget as HTMLTemplateElement;
+      const cloneTemplate = template.content.cloneNode(true) as HTMLElement;
+      const listElement = cloneTemplate.querySelector(
+        '[data-message-status="' + status + '"]',
```

Comment:
```suggestion
       `data-message-status='${status}']`,
```

The new template string syntax is nicer to read, best to use for strimg building.

Reply:
I tried it, didn't seems to work.

Reply:
Sorry, had the syntax wrong, this should work.

```
document.querySelector(`[data-message-status=${status || 'success'}]`)
```

Note: Falling back to success here, but you could do that above when you pull out the value from the event details.

Reply:
No, I mean I did try the backtik string way initially and it threw an exception similar to "invalid query selector" that's I had to go with this concatenation method.

Reply:
Would try again and let you know, proper exception of any occurs.

Reply:
Note: pulling out an element this way from the Template's DocumentFragment should be fine. 

As per https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment/querySelector